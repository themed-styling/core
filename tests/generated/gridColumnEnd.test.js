import 'jest-styled-components'

import { gridColumnEnd } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(gridColumnEnd, 'gridColumnEnd')
