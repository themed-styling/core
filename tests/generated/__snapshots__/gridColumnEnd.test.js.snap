// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gridColumnEnd renders 0 1`] = `
Array [
  .c0 {
  grid-column-end: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders function 1`] = `
Array [
  .c0 {
  grid-column-end: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: has;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: has;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders invalid theme path 1`] = `
Array [
  .c0 {
  grid-column-end: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders negative number 1`] = `
Array [
  .c0 {
  grid-column-end: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    grid-column-end: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-end: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    grid-column-end: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-end: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-end: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-end: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-end: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    grid-column-end: July;
  }
}

@media April {
  .c0 {
    grid-column-end: August;
  }
}

@media January {
  .c0 {
    grid-column-end: September;
  }
}

@media April {
  .c0 {
    grid-column-end: November;
  }
}

@media June {
  .c0 {
    grid-column-end: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    grid-column-end: July;
  }
}

@media April {
  .c0 {
    grid-column-end: August;
  }
}

@media January {
  .c0 {
    grid-column-end: September;
  }
}

@media April {
  .c0 {
    grid-column-end: November;
  }
}

@media June {
  .c0 {
    grid-column-end: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-column-end: January;
  }
}

@media April {
  .c0 {
    grid-column-end: December;
  }
}

@media January {
  .c0 {
    grid-column-end: April;
  }
}

@media April {
  .c0 {
    grid-column-end: May;
  }
}

@media June {
  .c0 {
    grid-column-end: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-column-end: January;
  }
}

@media April {
  .c0 {
    grid-column-end: December;
  }
}

@media January {
  .c0 {
    grid-column-end: April;
  }
}

@media April {
  .c0 {
    grid-column-end: May;
  }
}

@media June {
  .c0 {
    grid-column-end: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders postitive number 1`] = `
Array [
  .c0 {
  grid-column-end: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders string 1`] = `
Array [
  .c0 {
  grid-column-end: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: June;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: June;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnEnd renders valid theme path 1`] = `
Array [
  .c0 {
  grid-column-end: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-end: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
